pallets/parachain-staking/src/inflation.rs

```diff
@@ -145,7 +141,7 @@ mod tests {
 		let rate = Perquintill::one();
 		assert!(almost_equal(
 			rate * 10_000_000_000u128,
```

Comment:
Can we make this `10_000_000_000u128` a constant or a variable somehow instead of a magic constant?

Reply:
Ah ok these are tests, nevermind. 😄